dm/dm/config/task.go

```diff
@@ -337,7 +339,8 @@ func (m *SyncerConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 }
 
 type ValidatorConfig struct {
-	Mode string `yaml:"mode" toml:"mode" json:"mode"`
+	Mode        string `yaml:"mode" toml:"mode" json:"mode"`
+	WorkerCount int    `yaml:"worker-count" toml:"worker-count" json:"worker-count"`
```

Comment:
Please fix JSON tag name.

Reply:
you mean `-` to `_`?

most of dm code uses `-` actually, we should follow this convention

Reply:
Oops, the standard JSON tags should be `_`.

Reply:
JSON api recommend uses camelCase actually.

for dm, i think we should stick to current convention to make it consistent with current names.

Reply:
😭 In fact the new json tags added are already `_` like  [TaskCliArgs](https://github.com/pingcap/tiflow/blob/21e7eb610c16ec0d123aab5ca7290c3d2b3d6816/dm/dm/config/task_cli_args.go#L31)

I think it needs to be standardized to `_` at least when we need to expose the josn format to the user, which is what all `OpenAPI` currently do.

---
Convention over Configuration
_Who cares what format your database primary keys are described by? Does it really matter whether it’s “id”, “postId”, “posts_id”, or “pid”? Is this a decision that’s worthy of recurrent deliberation? No._

from [ruby on rails](https://rubyonrails.org/doctrine#convention-over-configuration)